misc/scratch/release.json

```diff
@@ -1,31 +1,29 @@
 {
-    "name": "chbench",
-    "launch_script": "MZ_WORKERS=4 bin/mzcompose --preserve-ports --find chbench run load-test",
+    "name": "kafka-ingest-open-loop",
+    "launch_script": "MZ_WORKERS=4 bin/mzcompose --preserve-ports --find kafka-ingest-open-loop run default --num-seconds=86400 --records-per-second=1000",
     "instance_type": "r5a.4xlarge",
     "ami": "ami-0b29b6e62f2343b46",
     "size_gb": 200,
     "tags": {
         "scrape_benchmark_numbers": "true",
-        "lt_name": "release-chbench",
+        "lt_name": "release-kafka-ingest-open-loop",
         "purpose": "load_test",
-        "mzconduct_workflow": "load-test",
-        "test": "chbench",
+        "test": "kafka-ingest-open-loop",
         "environment": "scratch"
     }
 }
 
 {
-    "name": "kinesis",
-    "launch_script": "MZ_WORKERS=4 bin/mzcompose --preserve-ports --find perf-kinesis run load-test",
+    "name": "kafka-ingest-open-loop-s3",
```

Comment:
I think to make it clear that these are persist vs nopersist (as opposed to something like persist-file vs persist-s3), I'd name this one "kafka-ingest-open-loop-persist". Given that S3 will be our production storage, we don't need to mention it

Reply:
done